src/plugins/intel_cpu/src/cpu_streams_calculation.hpp

```diff
@@ -0,0 +1,35 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+/**
+ * @file cpu_streams_calculation.hpp
+ * @brief A header file for CPU streams calulation implementation.
+ */
+
+#pragma once
+
+#include <vector>
+
+namespace ov {
+namespace intel_cpu {
+/**
+ * @brief      Generate streams information table according to processors type table
+ * @param[in]  input_streams is target streams set by user via NUM_STREAMS or hints.
+ *               - input "0" mean function generate the optimal number of streams
+ *               - LATENCY hint equals 1 stream.
```

Comment:
Is it true for multisocket systems? I assume there should be amount equal to number of Numa nodes.